scripts/apps/authoring/attachments/UploadAttachmentsModal.tsx

```diff
@@ -121,104 +114,120 @@ export class UploadAttachmentsModal extends React.PureComponent<IProps, IState>
-                                                        onChange={(value) => {
-                                                            this.updateItemMeta(index, 'title', value);
-                                                        }}
-                                                        disabled={this.state.saving}
-                                                    />
-                                                </div>
-                                                <div className="form__row">
-                                                    <Input
-                                                        type="text"
-                                                        label={gettext('Description')}
-                                                        required={true}
-                                                        value={item.meta.description}
-                                                        onChange={(value) => {
-                                                            this.updateItemMeta(index, 'description', value);
-                                                        }}
-                                                        disabled={this.state.saving}
-                                                    />
-                                                </div>
-                                                <div className="form__row">
-                                                    <Input
-                                                        type="text"
-                                                        label={gettext('File Name')}
-                                                        required={true}
-                                                        value={item.file.name}
-                                                        onChange={() => false}
-                                                        disabled={true}
-                                                    />
-                                                </div>
-                                                <div className="form__row">
-                                                    <Input
-                                                        type="text"
-                                                        label={gettext('File Size')}
-                                                        required={true}
-                                                        value={item.file.size.toString()}
-                                                        onChange={() => false}
-                                                        disabled={true}
-                                                    />
-                                                </div>
+            <Modal
+                visible
+                zIndex={1050}
+                size="x-large"
+                position="center"
+                onHide={this.state.saving ? null : this.props.closeModal}
```

Comment:
According to the interface, `onHide` is a mandatory function. So if we want to work around this, we should pass a function that is not doing anything(like `noop` from lodash) - and not `null`.